summary_workflows/quantification/quantification_dockers/q_consolidation/manage_assessment_data.py

```diff
@@ -1,30 +1,114 @@
 #!/usr/bin/env python3
 
 from __future__ import division
-import shutil
+import requests
 import json
 import os
-import fnmatch
+import logging
+import sys
+import datetime
 from argparse import ArgumentParser
-import numpy as np
 from assessment_chart import assessment_chart
 
-DEFAULT_eventMark = '2018-04-05'
+DEFAULT_OEB_API = "https://dev-openebench.bsc.es/api/scientific/graphql"
+DEFAULT_eventMark_id = "OEBE0070000000"
+METRICS =  {"correlation":"OEBM0070000000"} ## is this needed?
```

Comment:
Not sure whether METRICS is needed, as the parts of the code using it have been removed. I'll comment on the positions where METRICS is used in the TCGA workflow

Reply:
I'm checking this with a colleague but it seems that the `METRICS` dictionary and the `.datalink.inline_data.metrics` object in the aggregation file were added so that the metric id could be shown on the plots. I'll confirm on Tuesday but I'm rather sure that all the references to `METRICS` can be removed. Even more so considering that you commented those parts out and it kept working.

Reply:
I can confirm that all references to `METRICS` can be removed, there's no need for the mapping for now. It was just some new feature that a colleague was testing.